code/game/machinery/computer/tram_controls.dm

```diff
@@ -15,14 +17,35 @@
 	//find the tram, late so the tram is all... set up so when this is called? i'm seriously stupid and 90% of what i do consists of barely educated guessing :)
 	find_tram()
 
+/obj/machinery/computer/tram_controls/Destroy()
+	for(var/obj/structure/industrial_lift/tram/tram_finder in GLOB.lifts)
+		tram_finder.console = null
```

Comment:
use a `COMSIG_PARENT_QDELETING` here rather then searching the list. hell what happens if there's more then one tram per station?

Reply:
probably something not good because the code is written under the assumption there is only 1 active tram on the map

Reply:
depression